src/Form/BillingConfigForm.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Drupal\apigee_m10n\Form;
+
+use Drupal\apigee_m10n\Controller\PrepaidBalanceController;
+use Drupal\Core\Cache\Cache;
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Datetime\DateFormatterInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Config form for billing.
+ *
+ * @package Drupal\apigee_m10n
+ */
+class BillingConfigForm extends ConfigFormBase {
```

Comment:
Getting nit picky here but `BillingConfigForm` sounds very. similar to the `BillingDetailsForm`.

I think we want to create a monetization settings form with a cache section we can move this one setting to.

Reply:
I renamed the form to `PrepaidBalanceConfigForm`.